packages/contracts/migrations/development/Deploy001_Diamond_Dollar.s.sol

```diff
@@ -276,8 +290,9 @@ contract Deploy001_Diamond_Dollar is Script, DiamondTestHelper {
         // add collateral token (users can mint/redeem Dollars in exchange for collateral)
         uint256 poolCeiling = 10_000e18; // max 10_000 of collateral tokens is allowed
         ubiquityPoolFacet.addCollateralToken(
-            collateralTokenAddress,
-            poolCeiling
+            collateralTokenAddress, // collateral token address
+            address(chainLinkPriceFeedLusdUsd), // chainlink LUSD/USD price feed address
```

Comment:
i think we should just rely on the commenting on what the variable naming is for `(LUSD/USD pairing)` an instead an dcent `chainLinkPriceFeedLUSD` to avoid a refactor in another PR

Reply:
I don't understand what exactly should be changed

Reply:
i suggest to change `chainLinkPriceFeedLusdUsd` in favor of `chainLinkPriceFeedLUSD`

Reply:
nevermind i think it's fine since it looks like you are busy with #833

Reply:
fixed https://github.com/ubiquity/ubiquity-dollar/pull/855/commits/603b21cdb844632a094bd26cf54b1c56fc79c000